doc/main_window.rst

```diff
@@ -29,112 +29,168 @@ Overview
+
+     ==  ==================  ============
+     #   Track               Description
+     ==  ==================  ============
+     1   Top Track           Clips that will always be visible in front of other clips
+     2   Middle Track        Clips in the middle (might or might not be visible, depending on what is above them)
+     3   Bottom Track        Clips that will always be hidden behind anything on the tracks above
+     ==  ==================  ============
 
 .. _keyboard_shortcut_ref:
 
 Keyboard Shortcuts
 ------------------
-Here is a list of the default keyboard shortcuts supported by OpenShot. You can configure these shortcuts
-in the Preferences dialog. Learning a few of these shortcuts can save you a bunch of time!
-
-==================  ============
-Shortcut            Action
-==================  ============
-Ctrl+H              About OpenShot
-Ctrl+M              Add Marker
-Ctrl+Shift+T        Add Track
-Ctrl+W              Add to Timeline
-Ctrl+B              Animated Title
-Ctrl+P              Choose Profile
-Ctrl+C              Copy
-Delete              Delete Item
-Backspace           Delete Item
-Ctrl+D              Details View
-Ctrl+E              Export Video
-L                   Fast Forward
-F11                 Fullscreen
-Ctrl+F              Import Files
-Ctrl+End            Jump to End
-Ctrl+Home           Jump to Start
-Ctrl+N              New Project
-Right               Next Frame
-Ctrl+Right          Next Marker
-Ctrl+O              Open Project
-Ctrl+V              Paste
-Space               Play/Pause Toggle
-Up                  Play/Pause Toggle
-Down                Play/Pause Toggle
-K                   Play/Pause Toggle
-Ctrl+Shift+P        Preferences
-Left                Previous Frame
-Ctrl+Left           Previous Marker
-Ctrl+I              Properties
-Ctrl+Q              Quit
-Ctrl+Y              Redo
-J                   Rewind
-Ctrl+S              Save Project
-Ctrl+Shift+S        Save Project As...
-Ctrl+A              Select All
-Ctrl+Shift+A        Select None
-Ctrl+K              Slice All: Keep Both Sides
-Ctrl+L              Slice All: Keep Left Side
-Ctrl+J              Slice All: Keep Right Side
-Ctrl+G              Toggle Snapping
-Ctrl+X              Split Clip
-Ctrl+Shift+D        Thumbnail View
-Ctrl+T              Title Editor
-Ctrl+Z              Undo
-=                   Zoom In
---                   Zoom Out
-Ctrl+Scroll         Zoom In/Out
-Ctrl+Middle Button  Scroll Timeline
-==================  ============
+Here is a list of the default keyboard shortcuts supported by OpenShot. You can
+configure these shortcuts in the Preferences window, which is opened by selecting
+:menuselection:`Edit --> Preferences` from the OpenShot menu bar.
+(On macOS, choose :menuselection:`OpenShot Video Editor --> Preferences`.)
+Learning a few of these shortcuts can save you a bunch of time!
+
+.. list-table::
+   :header-rows: 1
+   :widths: 15 20
+
+   * - Shortcut
+     - Action
```

Comment:
Should we add a description for each of these? Just trying to imagine a novice, wanting to learn these shortcuts... would they even understand the meaning of some of these?